src/components/CustomMarker/getImage.js

```diff
@@ -0,0 +1,45 @@
+import art from 'assets/images/art.png';
+import dating from 'assets/images/dating.png';
+import food from 'assets/images/food.png';
+import football from 'assets/images/football.png';
+import movies from 'assets/images/movies.png';
+import music from 'assets/images/music.png';
+import politics from 'assets/images/politics.png';
+import series from 'assets/images/series.png';
+import world from 'assets/images/world.png';
```

Comment:
Actually I did the same on my target and later realize that topic images are inside the topicsList. You need to fetch topics from API and inside each topic, you will have its respective image (and you have to find in the topics list your target topic)

Reply:
![image](https://user-images.githubusercontent.com/8755889/79495815-8e59f380-7ffb-11ea-83d6-8bd7479c28cf.png)

In case you don't believe me, this is my commit with the change 😂